_posts/2022-08-08-july-rubygems-updates.md

```diff
@@ -0,0 +1,41 @@
+---
+title: July 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Central](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in July.
+
+# RubyGems News
+
+This month in RubyGems, we released new versions of RubyGems [3.3.18](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3318--2022-07-14), [3.3.19](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3319--2022-07-27) and Bundler [2.3.18](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2318-july-14-2022), [2.3.19](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2319-july-27-2022).
+
+The following is a non-exhaustive list of other improvements included in the above releases (see the changelog for further information):
+ 
+- fixed an issue with Bundler on Windows that allows the new `x64-mingw-ucrt`, the default on **Ruby 3.1** to work seamlessly with the existing `platforms DSL` - [#5655](https://github.com/rubygems/rubygems/pull/5655). 
+- improved the  `bundler/setup` speed by lazily checking for missing platform-specific variants if nothing else has changed - [#5695](https://github.com/rubygems/rubygems/pull/5695).
```

Comment:
I would also include a reference to https://github.com/rubygems/rubygems/pull/5546 here, and don't go into details of each speed up. Just say: "Improve performance of `bundler/setup`".